.github/workflows/push-registrar-image.yml

```
@@ -0,0 +1,50 @@
+name: Build and Push Docker Images
+
+on:
+  workflow_dispatch:
+
+  schedule:
+    - cron: "0 4 * * *"
```

Let's schedule it to run only on the weekdays and mention the default time zone which will be used i.e. UTC
```suggestion
    - cron: "0 4 * * 1-5" # UTC Time
```